README.md

```diff
@@ -130,4 +130,27 @@ Join the [Discord server](discord.gg/melkeydevhouse)! We're always around to hel
 
 You can also probably catch Melkey working live on [Twitch](twitch.tv/melkey)! 
 
-Pop in and say hi! ♡
\ No newline at end of file
+Pop in and say hi! ♡
+
+## License
+
+MIT License
+
+Copyright (c) [2023] [Melkeydev](https://github.com/Melkeydev)
+
+Permission is hereby granted, free of charge, to any person obtaining a copy
+of this software and associated documentation files (the "Software"), to deal
+in the Software without restriction, including without limitation the rights
+to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+copies of the Software, and to permit persons to whom the Software is
+furnished to do so, subject to the following conditions:
+
+The above copyright notice and this permission notice shall be included in all
+copies or substantial portions of the Software.
+
+THE SOFTWARE IS PROVIDED "AS IS," WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS
+FOR A PARTICULAR PURPOSE, AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR
+COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES, OR OTHER LIABILITY, WHETHER
+IN AN ACTION OF CONTRACT, TORT, OR OTHERWISE, ARISING FROM, OUT OF, OR IN
+CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
```

Comment:
Typically a license is under a different file.
It is then referenced in the README.
Check out this repo's license for a good example:
https://github.com/Melkeydev/go-blueprint/blob/main/LICENSE

Reply:
Update README.md + Added LICENSE file
I think the link in the README.md will be available after the PR is approved.
Correct me if I'm mistaken!